daemon.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/leoleovich/3djuggler/lib"
```

Comment:
Not sure if a separate package is necessary... Can we at leaste have a meaningful name which would play nicely with types/constants in the package? "lib.JobStatus" is sort of too generic.